### PR TITLE
update release notes for ios sdk version 1.1.0

### PR DESCRIPTION
**Pull request by ChenLev**

### CODE REVIEW

**Comment by tbedford** [comment]
Please note due to i18n changes this file has moved to: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md

File: _documentation/client-sdk/sdk-documentation/ios/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ navigation_weight: 0
...
 # Release Notes
 
+## 1.1.0 - 2019-11-14
+### Fixed
+- iOS 13 push notifications support
+- Start server call stability
+- Receiving a DTMF event in call and conversation
+
+### Added
+NXMConverastionDelegate did receive DTMF event method
```
Comment (pardel) [inline _documentation/client-sdk/sdk-documentation/ios/release-notes.md]: NXMConverastionDelegate should be NXMConversationDelegate

**Comment by tbedford** [comment]
@fabianrbz I'm not sure how to fix merge conflicts in forks. Can you provide instructions pls on how to resolve this safely? Thanks.

**Comment by fabianrbz** [comment]
@tbedford @ChenLev 
Copying the changes to `_documentation/en/client-sdk/sdk-documentation/ios/release-notes.md` and removing `_documentation/client-sdk/sdk-documentation/ios/release-notes.md` should do it.

**Pull request merged**